kinto/__init__.py

```diff
@@ -42,13 +42,17 @@ def main(global_config, config=None, **settings):
                        version=__version__,
                        default_settings=DEFAULT_SETTINGS)
 
-    config.include('kinto.plugins.default_bucket')
+    settings = config.get_settings()
+
+    # In Kinto API 1.x, a default bucket is available.
+    # Force its inclusion if not specified in settings.
+    if 'kinto.plugins.default_bucket' not in settings['includes']:
+        config.include('kinto.plugins.default_bucket')
```

Comment:
Should we issue a warning and ask for it to be included in the configuration explicitely now?